src/Extensions/Hosting.Services.Web/KestrelListenerBuilder.cs

```diff
@@ -64,9 +65,22 @@ private void ConfigureServices(TContext context, IServiceCollection services)
 		{
 			services.AddSingleton<ServiceContext>(context);
 			services.AddSingleton(context);
+			AddMiddlewares(services);
 		}
 
 		private readonly ServiceFabricIntegrationOptions m_options;
+
 		private readonly Action<IWebHostBuilder> m_builderExtension;
+
+		private static void AddMiddlewares(IServiceCollection services)
```

Comment:
I've tried that initially, but could not be an extension method because it's private and not in the static class, so the call would be a bit awkward ex: `AddMiddlewares(collection).Add...` and I've decided to just live it as a simple call since it's an only place where it's needed.